### ISSUE TITLE
Module `AccessibilityInfo` does not exist in the Haste module map

### ISSUE THREAD

**Issue by IOSTimor** [facebook/react-native#30153]
1.版本信息
react-native-cli: 2.0.1
react-native: 0.57.8

2.错误信息
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `/**/**/**/**/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.

**Comment by IOSTimor** [comment]
React Native Environment Info:
    System:
      OS: macOS 10.15.4
      CPU: (4) x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
      Memory: 137.80 MB / 8.00 GB
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 13.6, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 11.6/11E708 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.1 => 16.6.1 
      react-native: ^0.57.8 => 0.57.8 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-fetch-blob: 0.10.8
      react-native-gesture-handler: 1.6.1
      react-native-keep-awake: 4.0.0
      react-native-linear-gradient: 2.5.6
      react-native-pdf: 6.2.0
      react-native-qq: 2.0.5
      react-native-router-flux: 4.2.0
      react-native-svg: 12.1.0
      react-native-video: 4.4.5
      react-native-view-pdf: 0.11.0

**Comment by IOSTimor** [comment]
> ⚠️	Missing Environment Information
> ℹ️	Your issue may be missing information about your development environment. You can obtain the missing information by running `react-native info` in a console.

 React Native Environment Info:
    System:
      OS: macOS 10.15.4
      CPU: (4) x64 Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
      Memory: 137.80 MB / 8.00 GB
      Shell: 5.7.1 - /bin/zsh
    Binaries:
      Node: 10.16.0 - /usr/local/bin/node
      Yarn: 1.16.0 - /usr/local/bin/yarn
      npm: 6.9.0 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 13.6, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
    IDEs:
      Android Studio: 3.4 AI-183.6156.11.34.5522156
      Xcode: 11.6/11E708 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.1 => 16.6.1 
      react-native: ^0.57.8 => 0.57.8 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-fetch-blob: 0.10.8
      react-native-gesture-handler: 1.6.1
      react-native-keep-awake: 4.0.0
      react-native-linear-gradient: 2.5.6
      react-native-pdf: 6.2.0
      react-native-qq: 2.0.5
      react-native-router-flux: 4.2.0
      react-native-svg: 12.1.0
      react-native-video: 4.4.5
      react-native-view-pdf: 0.11.0

**Comment by safaiyeh** [comment]
Hi @IOSTimor your React Native version is pretty old. We no longer give support for it. I will close this issue, if you upgrade and this continues to happen please feel free to open a new issue.
